release-notes/10.0/supported-os.md

```diff
@@ -47,9 +47,9 @@ Notes:
 OS                              | Versions                    | Architectures         | Lifecycle
 ------------------------------- | --------------------------- | --------------------- | ----------------------
 [Alpine][6]                     | 3.21                        | Arm32, Arm64, x64     | [Lifecycle][7]
-[Azure Linux][8]                | 3.0                         | Arm64, x64            | None
```

Comment:
Seems like distroessed changed here? Shouldn't make a hard difference.

Reply:
Looks like some logic got lost in the tool. It needs to be updated. I can do that later. Good catch.

Reply:
It is fixed now.